install/installer/pkg/components/registry-facade/daemonset.go

```diff
@@ -189,6 +189,13 @@ func daemonset(ctx *common.RenderContext) ([]runtime.Object, error) {
 					RestartPolicy:                 corev1.RestartPolicyAlways,
 					TerminationGracePeriodSeconds: pointer.Int64(30),
 					InitContainers:                initContainers,
+					Tolerations: []corev1.Toleration{
+						{
+							Key:      "gitpod.io/gpu",
+							Operator: "Exists",
+							Effect:   "NoSchedule",
```

Comment:
nit: To my understanding we do not need the `Effect` here with operator `Exists`

From the [docs](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/#concepts):
> The default value for operator is Equal.
> A toleration "matches" a taint if the keys are the same and the effects are the same, and:
> - the operator is Exists (in which case no value should be specified), or
> - the operator is Equal and the values should be equal.

Reply:
@geropl 
I do not understand it this way. Just below of what you quoted (We only want to match `NoSchedule`)

> An empty effect matches all effects with key key1.

Reply:
>  An empty effect matches all effects with key key1.

Ja, that section is badly worded. I understood that this line only applies to `Equal`, but who knows :sweat_smile: 

> (We only want to match NoSchedule)

:bulb: Ah, did not know! Then it is the right thing I guess. :+1: